test/lifetime.ll

```diff
@@ -5,13 +5,12 @@
 ; RUN: llvm-spirv -r %t.spv -o %t.spv.bc
 ; RUN: llvm-dis < %t.spv.bc | FileCheck %s --check-prefix=CHECK-LLVM
 
-; CHECK-SPIRV-NOT: Bitcast
-; CHECK-SPIRV: 3 LifetimeStart [[tmp:[0-9]+]] 0
-; CHECK-SPIRV: 3 LifetimeStop [[tmp]] 0
+; CHECK-SPIRV: 3 LifetimeStart [[tmp:[0-9]+]] 4294967295
+; CHECK-SPIRV: 3 LifetimeStop [[tmp]] 4294967295
 
 ; CHECK-LLVM: %[[tmp1:[0-9]+]] = bitcast i32* %{{[0-9]+}} to i8*
-; CHECK-LLVM: call void @llvm.lifetime.start.p0i8(i64 -1, i8* %[[tmp1]])
-; CHECK-LLVM: call void @llvm.lifetime.end.p0i8(i64 -1, i8* %[[tmp1]])
+; CHECK-LLVM: call void @llvm.lifetime.start.p0i8(i64 4294967295, i8* %[[tmp1]])
```

Comment:
When we create the intrinsic during translation from SPIR-V to LLVM and the size is zero I think we can omit size parameter, i.e use default parameter value of CreateLifetimeStart function.